src/modules/DialogOverlay.vue

```diff
@@ -244,6 +257,36 @@ export default defineComponent({
 
       return this.appendUnits(orderedContent)
     },
+    setTerms(id: string) {
```

Comment:
`termValues` are created in line with - https://flexicards.atlassian.net/wiki/spaces/CO/pages/2299003254/Cards+Online+-+Terms+Conditions+Dynamic+Values
![image](https://user-images.githubusercontent.com/63985172/142331900-d24ced05-a058-4d3b-b395-203a6d93dce9.png)